Windows build problems related to fast matrix market parts

I think there are multiple things happening here but here is the first problem breaking build 

<details>

```bash
C:\Users\ilhan\Documents\GitHub\scipy [remove_cdflib ≡]> python dev.py build
>  meson setup C:\Users\ilhan\Documents\GitHub\scipy\build --prefix C:\Users\ilhan\Documents\GitHub\scipy\build-install
The Meson build system
Version: 1.3.0
Source dir: C:\Users\ilhan\Documents\GitHub\scipy
Build dir: C:\Users\ilhan\Documents\GitHub\scipy\build
Build type: native build
Project name: SciPy
Project version: 1.12.0.dev0
C compiler for the host machine: cc (gcc 13.2.0 "cc (Rev2, Built by MSYS2 project) 13.2.0")
C linker for the host machine: cc ld.bfd 2.41
C++ compiler for the host machine: c++ (gcc 13.2.0 "c++ (Rev2, Built by MSYS2 project) 13.2.0")
C++ linker for the host machine: c++ ld.bfd 2.41
Cython compiler for the host machine: cython (cython 3.0.5)
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program python3 found: YES (C:\Users\ilhan\AppData\Local\Programs\Python\Python310\python.exe)
Run-time dependency python found: YES 3.10
Program cython found: YES (C:\Users\ilhan\AppData\Local\Programs\Python\Python310\Scripts\cython.EXE)
Compiler for C supports arguments -Wno-unused-but-set-variable: YES
Compiler for C supports arguments -Wno-unused-function: YES
Compiler for C supports arguments -Wno-conversion: YES
Compiler for C supports arguments -Wno-misleading-indentation: YES
Library m found: YES
Fortran compiler for the host machine: gfortran (gcc 13.2.0 "GNU Fortran (Rev2, Built by MSYS2 project) 13.2.0")
Fortran linker for the host machine: gfortran ld.bfd 2.41
Compiler for Fortran supports arguments -Wno-conversion: YES
Checking if "-Wl,--version-script" : links: YES
Program pythran found: YES (C:\Users\ilhan\AppData\Local\Programs\Python\Python310\Scripts\pythran.EXE)
Found pkg-config: YES (C:\msys64\ucrt64\bin\pkg-config.EXE) 2.0.3
Did not find CMake 'cmake'
Found CMake: NO
Run-time dependency xsimd found: NO (tried pkgconfig and cmake)
Run-time dependency threads found: YES
Library npymath found: YES
Library npyrandom found: YES
pybind11-config found: YES (C:\Users\ilhan\AppData\Local\Programs\Python\Python310\Scripts\pybind11-config.EXE) 2.11.1
Run-time dependency pybind11 found: YES 2.11.1
Run-time dependency scipy-openblas found: NO (tried pkgconfig and cmake)
Run-time dependency openblas found: YES 0.3.24
Dependency openblas found: YES 0.3.24 (cached)
Compiler for C supports arguments -Wno-maybe-uninitialized: YES
Compiler for C supports arguments -Wno-discarded-qualifiers: YES
Compiler for C supports arguments -Wno-empty-body: YES
Compiler for C supports arguments -Wno-implicit-function-declaration: YES
Compiler for C supports arguments -Wno-parentheses: YES
Compiler for C supports arguments -Wno-switch: YES
Compiler for C supports arguments -Wno-unused-label: YES
Compiler for C supports arguments -Wno-unused-result: YES
Compiler for C supports arguments -Wno-unused-variable: YES
Compiler for C supports arguments -Wno-incompatible-pointer-types: YES
Compiler for C++ supports arguments -Wno-cpp: YES
Compiler for C++ supports arguments -Wno-deprecated-declarations: YES
Compiler for C++ supports arguments -Wno-class-memaccess: YES
Compiler for C++ supports arguments -Wno-format-truncation: YES
Compiler for C++ supports arguments -Wno-non-virtual-dtor: YES
Compiler for C++ supports arguments -Wno-sign-compare: YES
Compiler for C++ supports arguments -Wno-switch: YES
Compiler for C++ supports arguments -Wno-terminate: YES
Compiler for C++ supports arguments -Wno-unused-but-set-variable: YES
Compiler for C++ supports arguments -Wno-unused-function: YES
Compiler for C++ supports arguments -Wno-unused-local-typedefs: YES
Compiler for C++ supports arguments -Wno-unused-variable: YES
Compiler for C++ supports arguments -Wno-int-in-bool-context: YES
Compiler for Fortran supports arguments -Wno-argument-mismatch: YES
Compiler for Fortran supports arguments -Wno-conversion: YES (cached)
Compiler for Fortran supports arguments -Wno-intrinsic-shadow: YES
Compiler for Fortran supports arguments -Wno-maybe-uninitialized: YES
Compiler for Fortran supports arguments -Wno-surprising: YES
Compiler for Fortran supports arguments -Wno-uninitialized: YES
Compiler for Fortran supports arguments -Wno-unused-dummy-argument: YES
Compiler for Fortran supports arguments -Wno-unused-label: YES
Compiler for Fortran supports arguments -Wno-unused-variable: YES
Compiler for Fortran supports arguments -Wno-tabs: YES
Compiler for Fortran supports arguments -Wno-argument-mismatch: YES (cached)
Compiler for Fortran supports arguments -Wno-conversion: YES (cached)
Compiler for Fortran supports arguments -Wno-maybe-uninitialized: YES (cached)
Compiler for Fortran supports arguments -Wno-unused-dummy-argument: YES (cached)
Compiler for Fortran supports arguments -Wno-unused-label: YES (cached)
Compiler for Fortran supports arguments -Wno-unused-variable: YES (cached)
Compiler for Fortran supports arguments -Wno-tabs: YES (cached)
Checking if "Check atomic builtins without -latomic" : links: YES
Configuring __config__.py using configuration
Checking for function "open_memstream" : NO
Configuring messagestream_config.h using configuration
Compiler for Fortran supports arguments -w: YES
Checking for size of "void*" : 8
Compiler for Fortran supports arguments -w: YES (cached)
Checking for size of "void*" : 8 (cached)
Build targets in project: 250

SciPy 1.12.0.dev0

  User defined options
    prefix: C:\Users\ilhan\Documents\GitHub\scipy\build-install

Found ninja-1.11.1.git.kitware.jobserver-1 at C:\Users\ilhan\AppData\Local\Programs\Python\Python310\Scripts\ninja.EXE
Meson build setup OK
>  ninja -C C:\Users\ilhan\Documents\GitHub\scipy\build -j8
ninja: Entering directory `C:\Users\ilhan\Documents\GitHub\scipy\build'
[18/1610] Generating 'scipy\\_lib\\_ccallback_c.cp310-win_amd64.pyd.p\\_ccallback_c.c'
performance hint: C:\Users\ilhan\Documents\GitHub\scipy\scipy\_lib\_ccallback_c.pyx:138:82: Exception check will always require the GIL to be acquired. Declare the function as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.
performance hint: C:\Users\ilhan\Documents\GitHub\scipy\scipy\_lib\_ccallback_c.pyx:141:90: Exception check will always require the GIL to be acquired. Declare the function as 'noexcept' if you control the definition and you're sure you don't want the function to raise exceptions.
[269/1610] Compiling C object scipy/special/_ellip_harm_2.cp310-win_amd64.pyd.p/meson-generated__ellip_harm_2.c.obj
In file included from ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/ndarraytypes.h:1929,
                 from ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/ndarrayobject.h:12,
                 from ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/arrayobject.h:5,
                 from scipy/special/_ellip_harm_2.cp310-win_amd64.pyd.p/_ellip_harm_2.c:1196:
..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/npy_1_7_deprecated_api.h:14:9: note: '#pragma message: ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/npy_1_7_deprecated_api.h(14) : Warning Msg: Using deprecated NumPy API, disable it with #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION'
   14 | #pragma message(_WARN___LOC__"Using deprecated NumPy API, disable it with " \
      |         ^~~~~~~
[286/1610] Generating 'scipy\\special\\cython_special.cp310-win_amd64.pyd.p\\cython_special.c'

[...] snipped


[293/1610] Compiling C object scipy/special/_cdflib.cp310-win_amd64.pyd.p/meson-generated__cdflib.c.obj
scipy/special/_cdflib.cp310-win_amd64.pyd.p/_cdflib.c: In function '__Pyx_ImportType_3_0_5':
scipy/special/_cdflib.cp310-win_amd64.pyd.p/_cdflib.c:53990:13: warning: unknown conversion type character 'z' in format [-Wformat=]
53990 |             "%s.%s size changed, may indicate binary incompatibility. "
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
scipy/special/_cdflib.cp310-win_amd64.pyd.p/_cdflib.c:53991:24: note: format string is defined here
53991 |             "Expected %zd from C header, got %zd from PyObject",
      |                        ^
scipy/special/_cdflib.cp310-win_amd64.pyd.p/_cdflib.c:53990:13: warning: unknown conversion type character 'z' in format [-Wformat=]
53990 |             "%s.%s size changed, may indicate binary incompatibility. "
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
scipy/special/_cdflib.cp310-win_amd64.pyd.p/_cdflib.c:53991:47: note: format string is defined here
53991 |             "Expected %zd from C header, got %zd from PyObject",
      |                                               ^
scipy/special/_cdflib.cp310-win_amd64.pyd.p/_cdflib.c:53990:13: warning: too many arguments for format [-Wformat-extra-args]
53990 |             "%s.%s size changed, may indicate binary incompatibility. "
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


[...] snipped

   14 | #pragma message(_WARN___LOC__"Using deprecated NumPy API, disable it with " \
      |         ^~~~~~~
[1007/1610] Compiling C object scipy/stats/_rcont/rcont.cp310-win_amd64.pyd.p/meson-generated_rcont.c.obj
scipy/stats/_rcont/rcont.cp310-win_amd64.pyd.p/rcont.c: In function '__Pyx_ImportType_3_0_5':
scipy/stats/_rcont/rcont.cp310-win_amd64.pyd.p/rcont.c:26894:13: warning: unknown conversion type character 'z' in format [-Wformat=]
26894 |             "%s.%s size changed, may indicate binary incompatibility. "
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
scipy/stats/_rcont/rcont.cp310-win_amd64.pyd.p/rcont.c:26895:24: note: format string is defined here
26895 |             "Expected %zd from C header, got %zd from PyObject",
      |                        ^
scipy/stats/_rcont/rcont.cp310-win_amd64.pyd.p/rcont.c:26894:13: warning: unknown conversion type character 'z' in format [-Wformat=]
26894 |             "%s.%s size changed, may indicate binary incompatibility. "
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
scipy/stats/_rcont/rcont.cp310-win_amd64.pyd.p/rcont.c:26895:47: note: format string is defined here
26895 |             "Expected %zd from C header, got %zd from PyObject",
      |                                               ^
scipy/stats/_rcont/rcont.cp310-win_amd64.pyd.p/rcont.c:26894:13: warning: too many arguments for format [-Wformat-extra-args]
26894 |             "%s.%s size changed, may indicate binary incompatibility. "
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[1008/1610] Linking target scipy/stats/_rcont/rcont.cp310-win_amd64.pyd
Warning: corrupt .drectve at end of def file
[1009/1610] Compiling C++ object scipy/io/_fast_matrix_market/_fmm_core.cp310-win_amd64.pyd.p/src__fmm_core.cpp.obj
FAILED: scipy/io/_fast_matrix_market/_fmm_core.cp310-win_amd64.pyd.p/src__fmm_core.cpp.obj
"c++" "-Iscipy\io\_fast_matrix_market\_fmm_core.cp310-win_amd64.pyd.p" "-Iscipy\io\_fast_matrix_market" "-I..\scipy\io\_fast_matrix_market" "-I..\scipy\io\_fast_matrix_market\fast_matrix_market\include" "-I..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include" "-I..\scipy\io\_fast_matrix_market\fast_matrix_market\dependencies\fast_float\include" "-Iscipy\io\_fast_matrix_market\fast_matrix_market/dependencies/ryu" "-I..\scipy\io\_fast_matrix_market\fast_matrix_market\dependencies\ryu" "-IC:\Users\ilhan\AppData\Local\Programs\Python\Python310\Include" "-IC:\Users\ilhan\AppData\Local\Programs\Python\Python310\lib\site-packages\pybind11\include" "-fvisibility=hidden" "-fvisibility-inlines-hidden" "-fdiagnostics-color=always" "-D_FILE_OFFSET_BITS=64" "-Wall" "-Winvalid-pch" "-std=c++17" "-O2" "-g" "-D__USE_MINGW_ANSI_STDIO=1" "-DMS_WIN64=" "-DNPY_NO_DEPRECATED_API=NPY_1_9_API_VERSION" "-DFMM_SCIPY_PRUNE" "-DFMM_USE_FAST_FLOAT" "-DFMM_USE_RYU" "-DFMM_FROM_CHARS_INT_SUPPORTED" "-DFMM_TO_CHARS_INT_SUPPORTED" -MD -MQ scipy/io/_fast_matrix_market/_fmm_core.cp310-win_amd64.pyd.p/src__fmm_core.cpp.obj -MF "scipy\io\_fast_matrix_market\_fmm_core.cp310-win_amd64.pyd.p\src__fmm_core.cpp.obj.d" -o scipy/io/_fast_matrix_market/_fmm_core.cp310-win_amd64.pyd.p/src__fmm_core.cpp.obj "-c" ../scipy/io/_fast_matrix_market/src/_fmm_core.cpp
In file included from ../scipy/io/_fast_matrix_market/src/_fmm_core.cpp:5:
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:48:46: error: expected ')' before 'vector_length'
   48 |         explicit matrix_market_header(int64_t vector_length) : object(vector), vector_length(vector_length) {}
      |                                      ~       ^~~~~~~~~~~~~~
      |                                              )
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:49:37: error: expected ')' before 'nrows'
   49 |         matrix_market_header(int64_t nrows, int64_t ncols) : nrows(nrows), ncols(ncols) {}
      |                             ~       ^~~~~~
      |                                     )
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:57:9: error: 'int64_t' does not name a type
   57 |         int64_t nrows = 0;
      |         ^~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:9:1: note: 'int64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
    8 | #include <map>
  +++ |+#include <cstdint>
    9 | #include <string>
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:58:9: error: 'int64_t' does not name a type
   58 |         int64_t ncols = 0;
      |         ^~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:58:9: note: 'int64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:61:9: error: 'int64_t' does not name a type
   61 |         int64_t vector_length = 0;
      |         ^~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:61:9: note: 'int64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:64:9: error: 'int64_t' does not name a type
   64 |         int64_t nnz = 0;
      |         ^~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:64:9: note: 'int64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:70:9: error: 'int64_t' does not name a type
   70 |         int64_t header_line_count = 1;
      |         ^~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:70:9: note: 'int64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:80:9: error: 'int64_t' does not name a type
   80 |         int64_t chunk_size_bytes = 2 << 20;
      |         ^~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:80:9: note: 'int64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:132:9: error: 'int64_t' does not name a type
  132 |         int64_t chunk_size_values = 2 << 12;
      |         ^~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/types.hpp:132:9: note: 'int64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
In file included from ..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:9,
                 from ..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/fast_matrix_market.hpp:215,
                 from ../scipy/io/_fast_matrix_market/src/_fmm_core.cpp:17:
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/chunking.hpp: In function 'void fast_matrix_market::get_next_chunk(std::string&, std::istream&, const read_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/chunking.hpp:16:30: error: 'const struct fast_matrix_market::read_options' has no member named 'chunk_size_bytes'
   16 |         chunk.resize(options.chunk_size_bytes);
      |                              ^~~~~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/chunking.hpp: In function 'std::string fast_matrix_market::get_next_chunk(std::istream&, const read_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/chunking.hpp:53:35: error: 'const struct fast_matrix_market::read_options' has no member named 'chunk_size_bytes'
   53 |         std::string chunk(options.chunk_size_bytes, ' ');
      |                                   ^~~~~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp: In function 'int64_t fast_matrix_market::get_storage_nnz(const matrix_market_header&, read_options)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:73:27: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
   73 |             return header.nnz;
      |                           ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:78:35: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
   78 |                 return 2 * header.nnz;
      |                                   ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:80:31: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
   80 |                 return header.nnz;
      |                               ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:83:38: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
   83 |             auto diag_count = header.nrows;
      |                                      ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:84:42: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
   84 |             auto off_diag_count = header.nrows * header.ncols - diag_count;
      |                                          ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:84:57: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
   84 |             auto off_diag_count = header.nrows * header.ncols - diag_count;
      |                                                         ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:92:35: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
   92 |                     return header.nnz;
      |                                   ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:103:39: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  103 |                         return header.nnz;
      |                                       ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp: In function 'int64_t fast_matrix_market::read_header(std::istream&, matrix_market_header&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:222:49: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'vector_length'
  222 |                 pos = read_int(pos, end, header.vector_length);
      |                                                 ^~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:224:28: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'vector_length'
  224 |                 if (header.vector_length < 0) {
      |                            ^~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:230:53: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  230 |                     pos = read_int(pos, end, header.nnz);
      |                                                     ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:233:28: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  233 |                     header.nnz = header.vector_length;
      |                            ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:233:41: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'vector_length'
  233 |                     header.nnz = header.vector_length;
      |                                         ^~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:237:24: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  237 |                 header.nrows = header.vector_length;
      |                        ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:237:39: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'vector_length'
  237 |                 header.nrows = header.vector_length;
      |                                       ^~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:238:24: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
  238 |                 header.ncols = 1;
      |                        ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:240:49: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  240 |                 pos = read_int(pos, end, header.nrows);
      |                                                 ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:242:49: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
  242 |                 pos = read_int(pos, end, header.ncols);
      |                                                 ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:243:28: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  243 |                 if (header.nrows < 0 || header.ncols < 0) {
      |                            ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:243:48: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
  243 |                 if (header.nrows < 0 || header.ncols < 0) {
      |                                                ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:249:53: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  249 |                     pos = read_int(pos, end, header.nnz);
      |                                                     ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:250:32: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  250 |                     if (header.nnz < 0) {
      |                                ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:255:28: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  255 |                     header.nnz = header.nrows * header.ncols;
      |                            ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:255:41: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  255 |                     header.nnz = header.nrows * header.ncols;
      |                                         ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:255:56: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
  255 |                     header.nnz = header.nrows * header.ncols;
      |                                                        ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:258:37: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  258 |                 if (std::min(header.nrows, header.ncols) == 1) {
      |                                     ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:258:51: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
  258 |                 if (std::min(header.nrows, header.ncols) == 1) {
      |                                                   ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:260:28: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'vector_length'
  260 |                     header.vector_length = std::max(header.nrows, header.ncols);
      |                            ^~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:260:60: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  260 |                     header.vector_length = std::max(header.nrows, header.ncols);
      |                                                            ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:260:74: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
  260 |                     header.vector_length = std::max(header.nrows, header.ncols);
      |                                                                          ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:262:28: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'vector_length'
  262 |                     header.vector_length = -1;
      |                            ^~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:273:16: error: 'struct fast_matrix_market::matrix_market_header' has no member named 'header_line_count'
  273 |         header.header_line_count = lines_read;
      |                ^~~~~~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp: In function 'bool fast_matrix_market::write_header(std::ostream&, const matrix_market_header&, write_options)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:297:26: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'vector_length'
  297 |             os << header.vector_length;
      |                          ^~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:299:40: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  299 |                 os << kSpace << header.nnz;
      |                                        ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:302:26: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  302 |             os << header.nrows << kSpace << header.ncols;
      |                          ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:302:52: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
  302 |             os << header.nrows << kSpace << header.ncols;
      |                                                    ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/header.hpp:304:40: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  304 |                 os << kSpace << header.nnz;
      |                                        ^~~
In file included from ..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/fast_matrix_market.hpp:217:
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/formatters.hpp: In member function 'fast_matrix_market::triplet_formatter<LF, A_ITER, B_ITER, C_ITER>::chunk fast_matrix_market::triplet_formatter<LF, A_ITER, B_ITER, C_ITER>::next_chunk(const fast_matrix_market::write_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/formatters.hpp:172:48: error: 'const struct fast_matrix_market::write_options' has no member named 'chunk_size_values'
  172 |             auto chunk_size = std::min(options.chunk_size_values, (int64_t)(row_end - row_iter));
      |                                                ^~~~~~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/formatters.hpp: In member function 'fast_matrix_market::csc_formatter<LF, PTR_ITER, IND_ITER, VAL_ITER>::chunk fast_matrix_market::csc_formatter<LF, PTR_ITER, IND_ITER, VAL_ITER>::next_chunk(const fast_matrix_market::write_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/formatters.hpp:283:59: error: 'const struct fast_matrix_market::write_options' has no member named 'chunk_size_values'
  283 |             auto num_columns = (int64_t)(((double)options.chunk_size_values / nnz_per_column) + 1);
      |                                                           ^~~~~~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/formatters.hpp: In member function 'fast_matrix_market::dense_2d_call_formatter<LF, DenseType, DIM>::chunk fast_matrix_market::dense_2d_call_formatter<LF, DenseType, DIM>::next_chunk(const fast_matrix_market::write_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/formatters.hpp:407:54: error: 'const struct fast_matrix_market::write_options' has no member named 'chunk_size_values'
  407 |             auto num_columns = (DIM)((double)options.chunk_size_values / nrows) + 1;
      |                                                      ^~~~~~~~~~~~~~~~~
In file included from ..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/fast_matrix_market.hpp:218:
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp: In function 'fast_matrix_market::line_counts fast_matrix_market::read_chunk_matrix_coordinate(const std::string&, const matrix_market_header&, line_counts, HANDLER&, const read_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:228:48: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  228 |                 if (line.element_num >= header.nnz) {
      |                                                ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:242:68: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  242 |                 if (row <= 0 || static_cast<int64_t>(row) > header.nrows) {
      |                                                                    ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:245:68: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
  245 |                 if (col <= 0 || static_cast<int64_t>(col) > header.ncols) {
      |                                                                    ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp: In function 'fast_matrix_market::line_counts fast_matrix_market::read_chunk_vector_coordinate(const std::string&, const matrix_market_header&, line_counts, HANDLER&, const read_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:296:48: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  296 |                 if (line.element_num >= header.nnz) {
      |                                                ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:307:68: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'vector_length'
  307 |                 if (row <= 0 || static_cast<int64_t>(row) > header.vector_length) {
      |                                                                    ^~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp: In function 'fast_matrix_market::line_counts fast_matrix_market::read_chunk_array(const std::string&, const matrix_market_header&, line_counts, HANDLER&, const read_options&, typename HANDLER::coordinate_type&, typename HANDLER::coordinate_type&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:340:48: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  340 |             if (row == 0 && col == 0 && header.nrows > 0) {
      |                                                ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:358:57: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'ncols'
  358 |                 if (static_cast<int64_t>(col) >= header.ncols) {
      |                                                         ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:373:57: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  373 |                 if (static_cast<int64_t>(row) == header.nrows) {
      |                                                         ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:384:68: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  384 |                             if (static_cast<int64_t>(row) < header.nrows-1) {
      |                                                                    ^~~~~
In file included from ..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:407:
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body_threads.hpp: In function 'fast_matrix_market::line_counts fast_matrix_market::read_body_threads(std::istream&, const matrix_market_header&, HANDLER&, const read_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body_threads.hpp:57:31: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'header_line_count'
   57 |         line_counts lc{header.header_line_count, 0};
      |                               ^~~~~~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body_threads.hpp:114:41: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  114 |             if (lc.element_num > header.nnz) {
      |                                         ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body_threads.hpp:121:85: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  121 |                     typename HANDLER::coordinate_type row = lc.element_num % header.nrows;
      |                                                                                     ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body_threads.hpp:122:85: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nrows'
  122 |                     typename HANDLER::coordinate_type col = lc.element_num / header.nrows;
      |                                                                                     ^~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp: In function 'fast_matrix_market::line_counts fast_matrix_market::read_coordinate_body_sequential(std::istream&, const matrix_market_header&, HANDLER&, const read_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:414:31: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'header_line_count'
  414 |         line_counts lc{header.header_line_count, 0};
      |                               ^~~~~~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp: In function 'fast_matrix_market::line_counts fast_matrix_market::read_array_body_sequential(std::istream&, const matrix_market_header&, HANDLER&, const read_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:439:31: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'header_line_count'
  439 |         line_counts lc{header.header_line_count, 0};
      |                               ^~~~~~~~~~~~~~~~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp: In function 'void fast_matrix_market::read_matrix_market_body_no_adapters(std::istream&, const matrix_market_header&, HANDLER&, const read_options&)':
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:510:37: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  510 |         if (lc.element_num < header.nnz) {
      |                                     ^~~
..\scipy\io\_fast_matrix_market\fast_matrix_market\include/fast_matrix_market/read_body.hpp:513:56: error: 'const struct fast_matrix_market::matrix_market_header' has no member named 'nnz'
  513 |                                  std::to_string(header.nnz - lc.element_num) + " lines.");
      |                                                        ^~~

[...] snipped 

../scipy/io/_fast_matrix_market/src/_fmm_core.cpp:207:57: error: 'ncols' is not a member of 'fast_matrix_market::matrix_market_header'
  207 |     .def_readwrite("ncols", &fmm::matrix_market_header::ncols)
      |                                                         ^~~~~
../scipy/io/_fast_matrix_market/src/_fmm_core.cpp:209:55: error: 'nnz' is not a member of 'fast_matrix_market::matrix_market_header'
  209 |     .def_readwrite("nnz", &fmm::matrix_market_header::nnz)
      |                                                       ^~~
[1010/1610] Compiling C object scipy/io/matlab/_mio5_utils.cp310-win_amd64.pyd.p/meson-generated__mio5_utils.c.obj
In file included from ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/ndarraytypes.h:1929,
                 from ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/ndarrayobject.h:12,
                 from ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/arrayobject.h:5,
                 from scipy/io/matlab/_mio5_utils.cp310-win_amd64.pyd.p/_mio5_utils.c:1211:
..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/npy_1_7_deprecated_api.h:14:9: note: '#pragma message: ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/npy_1_7_deprecated_api.h(14) : Warning Msg: Using deprecated NumPy API, disable it with #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION'
   14 | #pragma message(_WARN___LOC__"Using deprecated NumPy API, disable it with " \
      |         ^~~~~~~
[1011/1610] Compiling C object scipy/stats/_unuran/unuran_wrapper.cp310-win_amd64.pyd.p/meson-generated_unuran_wrapper.c.obj
In file included from ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/ndarraytypes.h:1929,
                 from ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/ndarrayobject.h:12,
                 from ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/arrayobject.h:5,
                 from scipy/stats/_unuran/unuran_wrapper.cp310-win_amd64.pyd.p/unuran_wrapper.c:1192:
..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/npy_1_7_deprecated_api.h:14:9: note: '#pragma message: ..\..\..\..\AppData\Local\Programs\Python\Python310\lib\site-packages\numpy\core\include/numpy/npy_1_7_deprecated_api.h(14) : Warning Msg: Using deprecated NumPy API, disable it with #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION'
   14 | #pragma message(_WARN___LOC__"Using deprecated NumPy API, disable it with " \
      |         ^~~~~~~
[1016/1610] Compiling C++ object scipy/fft/_pocketfft/pypocketfft.cp310-win_amd64.pyd.p/pypocketfft.cxx.obj
ninja: build stopped: subcommand failed.
Build failed!
```

</details>

Secondly, either windows or the tools we use for the emojis are also broken for me 

```bash
C:\Users\ilhan\Documents\GitHub\scipy [remove_cdflib ≡]> python dev.py
                                                                                                                                                                                                                                                                                          
 ←[33mUsage:←[0m ←[1mdev.py←[0m [←[1;36mOPTIONS←[0m] ←[1;36mCOMMAND←[0m [←[1;36mARGS←[0m]...                                                                                                                                                                                              
                                                                                                                                                                                                                                                                                          
 Developer Tool for SciPy                                                                                                                                                                                                                                                                 
 ←[2mCommands that require a built/installed instance are marked with 🔧.←[0m                                                                                                                                                                                                             
                                                                                                                                                                                                                                                                                          
 ←[1;2mpython dev.py --build-dir my-build test -s stats←[0m                                                                                                                                                                                                                               
                                                                                                                                                                                                                                                                                          
←[2m╭─←[0m←[2m Options ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────←[0m←[2m─╮←[0m
←[2m│←[0m ←[1;36m-←[0m←[1;36m-help←[0m                ←[1;33m           ←[0m  Show this message and exit.                                                                                                                                                                                                                         ←[2m│←[0m
←[2m│←[0m ←[1;36m-←[0m←[1;36m-build←[0m←[1;36m-dir←[0m           ←[1;33mBUILD_DIR  ←[0m  🔧 Relative path to the build directory.                                                                                                                                                                                                            ←[2m│←[0m
←[2m│←[0m                                    ←[2m[default: build]                                                                                                                                                                                                                                   ←[0m ←[2m│←[0m
←[2m│←[0m ←[1;36m-←[0m←[1;36m-no←[0m←[1;36m-build←[0m        ←[1;32m-n←[0m  ←[1;33m           ←[0m  🔧 Do not build the project (note event python only modification require build).                                                                                                                                                                    ←[2m│←[0m
←[2m│←[0m ←[1;36m-←[0m←[1;36m-install←[0m←[1;36m-prefix←[0m      ←[1;33mINSTALL_DIR←[0m  🔧 Relative path to the install directory. Default is <build-dir>-install.                                                                                                                                                                          ←[2m│←[0m
←[2m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯←[0m
←[2m╭─←[0m←[2m build & testing ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────←[0m←[2m─╮←[0m
←[2m│←[0m ←[1;36mbuild←[0m←[1;36m ←[0m 🔧 Build & install package on path.                                                                                                                                                                                                                                             ←[2m│←[0m
←[2m│←[0m ←[1;36mtest ←[0m←[1;36m ←[0m 🔧 Run tests.                                                                                                                                                                                                                                                                   ←[2m│←[0m
←[2m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯←[0m
←[2m╭─←[0m←[2m static checkers ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────←[0m←[2m─╮←[0m
←[2m│←[0m ←[1;36mlint←[0m←[1;36m ←[0m 💨 Run linter on modified files and check for disallowed Unicode characters and possibly-invalid test names.                                                                                                                                                                     ←[2m│←[0m
←[2m│←[0m ←[1;36mmypy←[0m←[1;36m ←[0m 🔧 Run mypy on the codebase.                                                                                                                                                                                                                                                     ←[2m│←[0m
←[2m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯←[0m
←[2m╭─←[0m←[2m environments ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────←[0m←[2m─╮←[0m
←[2m│←[0m ←[1;36mshell          ←[0m←[1;36m ←[0m 🔧 Start Unix shell with PYTHONPATH set.                                                                                                                                                                                                                              ←[2m│←[0m
←[2m│←[0m ←[1;36mpython         ←[0m←[1;36m ←[0m 🔧 Start a Python shell with PYTHONPATH set.                                                                                                                                                                                                                          ←[2m│←[0m
←[2m│←[0m ←[1;36mipython        ←[0m←[1;36m ←[0m 🔧 Start IPython shell with PYTHONPATH set.                                                                                                                                                                                                                           ←[2m│←[0m
←[2m│←[0m ←[1;36mshow_PYTHONPATH←[0m←[1;36m ←[0m ℹ Show value of the PYTHONPATH environment variable used in this script.                                                                                                                                                                                              ←[2m│←[0m
←[2m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯←[0m
←[2m╭─←[0m←[2m documentation ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────←[0m←[2m─╮←[0m
←[2m│←[0m ←[1;36mdoc           ←[0m←[1;36m ←[0m 🔧 Build documentation.                                                                                                                                                                                                                                                ←[2m│←[0m
←[2m│←[0m ←[1;36mrefguide-check←[0m←[1;36m ←[0m 🔧 Run refguide check.                                                                                                                                                                                                                                                 ←[2m│←[0m
←[2m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯←[0m
←[2m╭─←[0m←[2m release ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────←[0m←[2m─╮←[0m
←[2m│←[0m ←[1;36mnotes  ←[0m←[1;36m ←[0m 📒 Release notes and log generation.                                                                                                                                                                                                                                          ←[2m│←[0m
←[2m│←[0m ←[1;36mauthors←[0m←[1;36m ←[0m 📒 Generate list of authors who contributed within revision interval.                                                                                                                                                                                                         ←[2m│←[0m
←[2m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯←[0m
←[2m╭─←[0m←[2m benchmarking ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────←[0m←[2m─╮←[0m
←[2m│←[0m ←[1;36mbench←[0m←[1;36m ←[0m 🔧 Run benchmarks.                                                                                                                                                                                                                                                              ←[2m│←[0m
←[2m╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯←[0m

```

don't know how to go about it. 

<strike>Third one is that the option that used to work `--win-cp-openblas` is now not recognized.</strike> OK nevermind I found the new one.

--- a/scipy/io/_fast_matrix_market/fast_matrix_market/include/fast_matrix_market/types.hpp
+++ b/scipy/io/_fast_matrix_market/fast_matrix_market/include/fast_matrix_market/types.hpp
@@ -6,6 +6,7 @@
 
 #include <complex>
 #include <map>
+#include <cstdint>
 #include <string>
 
 namespace fast_matrix_market {

